### DISCUSSION TITLE
Request of new code in "Reserved procurement" controlled vocabulary (PPDS)

### DISCUSSION THREAD

**Discussion by acolomer** [eprocurementontology/eprocurementontology#527] [Q&A]
Working in the PPDS project, we have noticed that the EU Vocabularies code list "Reserved procurement" does not include the code / concept "PARTICULAR_PROFESSION", which is used in the data coming from Austria.
Could this code / concept be added in "Reserved procurement" controlled vocabulary?
Otherwise, how should we map this code to ePO?

**Comment by AchillesDougalis** [comment]
To add a code to the codelist, we need a clear definition of the concept. If the data coming from Austria defines the particular profession we would suggest using a predicate such as _isReservedForAParticularProfession_ and then using the code list Occupation.  Could you please let us know how the data is used in Austria?

**Comment by acolomer** [comment]
Thank you very much for your reply.
We've seen that in the mapping for TED, the code / concept "PARTICULAR_PROFESSION" is mapped with [https://publications.europa.eu/resource/authority/selection-criterion/sui-act](https://publications.europa.eu/resource/authority/selection-criterion/sui-act).
Therefore, if this is correct, it will not be necessary to add a new code.
Thank you.
